### PR TITLE
feat: set, get function for aliasname

### PR DESCRIPTION
**Pull request by hcrypto7**

### CODE REVIEW

File: src/core/libs/FactoryLib.sol [hunk 1]
```diff
@@ -80,6 +80,23 @@ library FactoryLib {
         uint len;
     }
 
+    // Mapping from address to alias name
+    mapping(address => string) public aliasNames;
```
Comment (a17) [inline src/core/libs/FactoryLib.sol:84]: * this mapping must be in Factory ERC7201 storage
* use named parameters in mapping types (Solidity 0.8.18+)

File: src/core/libs/FactoryLib.sol [hunk 2]
```diff
@@ -80,6 +80,23 @@ library FactoryLib {
...
+
+    // Event to log changes in the alias name
+    event AliasNameChanged(address indexed operator, address indexed tokenAddress, string newAliasName);
```
Comment (a17) [inline src/core/libs/FactoryLib.sol:87]: put event to IFactory like other events

Comment (hcrypto7) [reply src/core/libs/FactoryLib.sol]: fixed problems

File: src/core/Factory.sol [hunk 3]
```diff
@@ -22,7 +23,7 @@ import "../interfaces/IStrategyLogic.sol";
 /// @notice Platform factory assembling vaults. Stores vault settings, strategy logic, farms.
 ///         Provides the opportunity to upgrade vaults and strategies.
 /// Changelog:
-///   1.1.0: getDeploymentKey fix for not farming strategies, strategyAvailableInitParams
```
Comment (a17) [inline src/core/Factory.sol:25]: dont delete changelog

File: src/core/Factory.sol [hunk 4]
```diff
@@ -33,7 +34,7 @@ contract Factory is Controllable, ReentrancyGuardUpgradeable, IFactory {
     //region ----- Constants -----
 
     /// @inheritdoc IControllable
-    string public constant VERSION = "1.1.0";
+    string public constant VERSION = "1.1.1";
```
Comment (a17) [inline src/core/Factory.sol:37]: such amount of important changes must increase MINOR version, not PATCH. set 1.2.0

File: src/core/Factory.sol [hunk 5]
```diff
@@ -22,7 +23,7 @@ import "../interfaces/IStrategyLogic.sol";
 /// @notice Platform factory assembling vaults. Stores vault settings, strategy logic, farms.
 ///         Provides the opportunity to upgrade vaults and strategies.
 /// Changelog:
-///   1.1.0: getDeploymentKey fix for not farming strategies, strategyAvailableInitParams
+///   1.1.1: reduced factory size. moved upgradeStrategyProxy, upgradeVaultProxy logic to FactoryLib
 /// @author Alien Deployer (https://github.com/a17)
 /// @author Jude (https://github.com/iammrjude)
 /// @author JodsMigel (https://github.com/JodsMigel)
```
Comment (a17) [inline src/core/Factory.sol:29]: add your author tag

File: test/core/Factory.t.sol [hunk 6]
```diff
@@ -13,6 +13,8 @@ import "../../src/test/MockVaultUpgrade.sol";
 import "../../src/test/MockERC721.sol";
 import "../base/MockSetup.sol";
 import "../../src/interfaces/IStrategyLogic.sol";
+import "../../chains/PolygonLib.sol";
+import "forge-std/console.sol";
```
Comment (a17) [inline test/core/Factory.t.sol:17]: console was already imported above

File: src/interfaces/IFactory.sol [hunk 7]
```diff
@@ -52,6 +52,7 @@ interface IFactory {
     event NewFarm(Farm[] farms);
     event UpdateFarm(uint id, Farm farm);
     event SetStrategyAvailableInitParams(string id, address[] initAddresses, uint[] initNums, int24[] initTicks);
+    event AliasNameChanged(address indexed operator, address indexed tokenAddress, string newAliasName);
```
Comment (a17) [inline src/interfaces/IFactory.sol:56]: add your author tag to interface natspec

**Review by a17 — CHANGES_REQUESTED** [review]

File: src/interfaces/IFactory.sol (hunk 7 above)
Comment (hcrypto7) [reply src/interfaces/IFactory.sol:57]: thank you for your review. I have fixed the issues.

**Review by a17 — APPROVED** [review]

**Pull request merged**